src/exports.jl

```diff
@@ -358,6 +360,7 @@ export
   # physics/fermions.jl
   fparity,
   isfermionic,
+  parity_sign,
```

Comment:
What's the reason behind adding this as a new export?

Reply:
Agreed – I'll just use `ITensors.parity_sign` for the one place where it gets called from `ITensorMPS`.